cortex-installer.sh

```diff
@@ -1925,6 +1928,24 @@ function ask_sudo() {
   fi
 }
 
+function prompt_for_usage_stats() {
+  if [ "$CORTEX_ENABLE_TELEMETRY" != "true" ] && [ "$CORTEX_ENABLE_TELEMETRY" != "false" ]; then
+    while true
+    do
+      echo
+      read -p "Would you like to help improve Cortex by anonymously sending errors and usage stats to the dev team [Y/n] " -n 1 -r
```

Comment:
Do you know why the convention is for "Y" to be capital and "n" lower case in the prompt? We accept upper or lower case, and don't have a default, so if it's one of those reasons maybe it should be "[y/n]". We can also update them in a separate commit.